hathor/transaction/resources/mempool.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2021 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from itertools import islice
+from typing import TYPE_CHECKING
+
+from twisted.web import resource
+
+from hathor.api_util import set_cors
+from hathor.cli.openapi_files.register import register_resource
+
+if TYPE_CHECKING:
+    from twisted.web.http import Request
+
+    from hathor.manager import HathorManager
+
+
+@register_resource
+class MempoolResource(resource.Resource):
+    """ Implements a web server API to return transactions on the mempool.
+
+    You must run with option `--status <PORT>`.
+    """
+    isLeaf = True
+
+    def __init__(self, manager: 'HathorManager'):
+        # Important to have the manager so we can know the tx_storage
+        self.manager = manager
+
+    def render_GET(self, request: 'Request') -> bytes:
+        """ Get request /mempool/ that returns transactions on the mempool
+
+            :rtype: string (json list)
+        """
+        request.setHeader(b'content-type', b'application/json; charset=utf-8')
+        set_cors(request, 'GET')
+
+        # Get a list of all txs on the mempool
+        tx_ids = map(
+            # get only tx_ids
+            lambda tx: tx.hash_hex,
+            # order by timestamp
+            sorted(list(self.manager.tx_storage.iter_mempool()), key=lambda tx: tx.timestamp),
+        )
+        # Only return up to 100 txs per call
+        data = {'success': True, 'transactions': list(islice(tx_ids, 100))}
```

Comment:
This 100 should be a parameter in our config file. And it would be good to have this in a test. Have 101 txs in the mempool and return only 100.

Reply:
Done.